package-build.el

```diff
@@ -769,7 +781,10 @@ Optionally PRETTY-PRINT the data."
               (lambda (elt)
                 (unless (symbolp (car elt))
                   (error "Invalid package name in dependency: %S" (car elt)))
-                (list (car elt) (version-to-list (cadr elt))))
+                (list
```

Comment:
I don't actually think this line should use the new regexp. The regexp is for *our* packages but the dependencies shouldn't be effected by the regexp.